synapseclient/api/json_schema_services.py

```diff
@@ -0,0 +1,224 @@
+import json
+from typing import TYPE_CHECKING, AsyncGenerator, Dict, List, Optional, Union
+
+from synapseclient.api.api_client import rest_post_paginated_async
+
+if TYPE_CHECKING:
+    from synapseclient import Synapse
+
+
+async def bind_json_schema_to_entity(
+    synapse_id: str,
+    json_schema_uri: str,
+    *,
+    enable_derived_annos: bool = False,
```

Comment:
set this as an optional parameter. This parameter should probably also be added to the sync function